src/routes/auth.ts

```diff
@@ -4,9 +4,10 @@ import * as jwt from 'express-jwt'
 const secret = process.env.SESSION_SECRET
 
 function getTokenFromHeader(req: Request) {
-  const auth = req.headers.authorization &&
+  const auth =
+    req.headers.authorization &&
     (req.headers.authorization as string).split(' ')
-  if (auth && auth[0] === 'Token' || auth[0] === 'Bearer') {
+  if ((auth && auth[0] === 'Token') || auth[0] === 'Bearer') {
```

Comment:
C'est pas bon avec les parenthèses en +.
Le premier check `auth` permet de vérifier que `auth` n'est pas undefined (ce qui peut arriver dans le cas où le header n'est pas set).
Dans ce cas là, auth[0] === 'Bearer' va throw une exception (`undefined is not an array`) par que ça passe le 1er check.

Reply:
Woups je pense que c'est prettier qui à ajouté les parenthèses pcq je l'ai pas fait mdr